src/database/database.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  addPouchPlugin,
+  addRxPlugin,
+  createRxDatabase,
+  getRxStoragePouch,
+  RxDatabase,
+} from "rxdb";
+import { RxDBLeaderElectionPlugin } from "rxdb/plugins/leader-election";
+import { RxDBReplicationCouchDBPlugin } from "rxdb/plugins/replication-couchdb";
+import { collectionsInit } from "./collections";
+
+addPouchPlugin(require("pouchdb-adapter-idb"));
+addPouchPlugin(require("pouchdb-adapter-http"));
+addRxPlugin(RxDBLeaderElectionPlugin);
+addRxPlugin(RxDBReplicationCouchDBPlugin);
+
+const syncURL = "http://" + window.location.hostname + ":10102/";
+console.log("host: " + syncURL);
+
+let dbPromise: any = null;
+
+const _create = async (): Promise<RxDatabase> => {
+  console.log("DatabaseService: creating database...");
+  const db = await createRxDatabase({
+    name: "ymocktestdb",
+    storage: getRxStoragePouch("idb"),
+  });
+  console.log("DatabaseService: created database");
+  window["db"] = db;
```

Comment:
Is it really necessary? If not, I would remove the global reference.

Reply:
This should be usefull in order to debug RxDB instance locally, If you prefer we can remove it.